org.codecoop.kjo.bookletimposer.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, the file should be contributed upstream.

Reply:
Yes sure but for now , i don't think the developer is initered in upstream this , last time i talk he say he maintain only for debian that why i made a flatpak ^^